src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java

```diff
@@ -64,7 +67,15 @@ protected Object doWork() {
         final File indexFile = determineFileName(index);
 
         try{
-            IndexFactory.writeIndex(index, indexFile);
+            /*
+             * Note: IndexFactory.writeIndex is incorrect for tabix indexes (does not use BlockCompressedOutputStream)
```

Comment:
Can you patch this in htsjdk (or, if you don't have time, could you at least create a ticket to fix this)?